### DISCUSSION TITLE
Transactions

### DISCUSSION THREAD

**Discussion by technic** [adelsz/pgtyped#448] [Q&A] [answered]
How to do transactions with pg-typed?

**Comment by adelsz** [comment, marked as answer]
Transactions can be run as usual in [node-postgres](https://node-postgres.com/features/transactions):  
```ts
await client.query('BEGIN')
const books = await findBookById.run({ bookId: 42 }, client);
await client.query('COMMIT')
```

**Reply by technic** [reply]
Ok, and I think I need some sort of wraper to `REJECT` when exception is thrown.
